server/routes/nl/api.py

```diff
@@ -223,11 +223,11 @@ def feedback():
       not current_app.config['LOG_QUERY']):
     flask.abort(404)
 
-  session_info = request.json('session_info')
-  feedback_data = request.json['feedback_data']
+  session_id = request.json['sessionId']
+  feedback_data = request.json['feedbackData']
```

Comment:
nit: Are you particular about camelCase?  Elsewhere the session-id uses snakecase (https://github.com/datacommonsorg/website/blob/b0331c234c77f303706cc5f9ac55faf197c4dae9/server/lib/nl/fulfillment/context.py#L139-L155)

Reply:
Json field should use camelCase..